app/forms/schools/add_participants/base_wizard.rb

```diff
@@ -462,19 +463,26 @@ def participant_cohort
 
       # NOTE: not preventing registration here just determining where to put the participant
       def cohort_to_place_participant
-        if transfer?
-          existing_participant_cohort || existing_participant_profile&.schedule&.cohort
-        elsif ect_participant? && induction_start_date.present?
-          Cohort.containing_date(induction_start_date).tap do |cohort|
-            return Cohort.current if cohort.blank? || cohort.npq_plus_one_or_earlier?
-          end
-        elsif Cohort.within_automatic_assignment_period?
-          Cohort.current
-        else
-          Cohort.next
+        return cohort_for_transfer if transfer?
+        return cohort_for_ect_with_induction_start_date if ect_participant? && induction_start_date.present?
+
+        Cohort.within_automatic_assignment_period? ? Cohort.current : Cohort.next
```

Comment:
```suggestion
        return Cohort.current if Cohort.within_automatic_assignment_period?
        
        Cohort.next
```

Reply:
If we are refactoring into short-circuit return statements (I prefer this for readability) perhaps taking the last ternary out is worthwhile for consistency.